Reboot tests fail for kubernetes-e2e-gke-ci-reboot (node doesn't come back after reboot)

```
Reboot [Disruptive] each node by triggering kernel panic and ensure they function upon restart

/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/reboot.go:102 Jan 22 00:17:06.634: Test failed; at least one node failed to reboot in the time given.


Reboot [Disruptive] each node by ordering unclean reboot and ensure they function upon restart

/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/reboot.go:96 Jan 22 00:24:12.487: Test failed; at least one node failed to reboot in the time given.
```

I made some investigation. The problem with crashlooping Heapster is due to:
```
I0125 19:34:39.971189       1 heapster.go:60] /heapster --source=kubernetes:''
I0125 19:34:39.972094       1 heapster.go:61] Heapster version 0.18.4
I0125 19:34:39.972911       1 kube_factory.go:169] Using Kubernetes client with master "https://10.51.240.1:443" and version "v1"
I0125 19:34:39.972962       1 kube_factory.go:170] Using kubelet port 10255
I0125 19:34:39.977155       1 heapster.go:71] Starting heapster on port 8082
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x68 pc=0x581450]

goroutine 115 [running]:
k8s.io/heapster/sources.(*kubeNodeMetrics).updateStats(0xc8201ad920, 0xc8200fa6c0, 0x20, 0xc8200151a0, 0xe, 0xc820015180, 0xa, 0xc8200fa860, 0x14, 0x3e8, ...)
        /usr/local/google/home/pszczesniak/go/src/k8s.io/heapster/sources/kube_nodes.go:86 +0xce0
k8s.io/heapster/sources.(*kubeNodeMetrics).getNodesInfo.func1(0xc820203260, 0xc8201ad920, 0xece386fd5, 0x0, 0x1215440, 0xece386fda, 0x0, 0x1215440, 0xc820203240, 0xc82043ea60, ...)
        /usr/local/google/home/pszczesniak/go/src/k8s.io/heapster/sources/kube_nodes.go:103 +0x130
created by k8s.io/heapster/sources.(*kubeNodeMetrics).getNodesInfo
        /usr/local/google/home/pszczesniak/go/src/k8s.io/heapster/sources/kube_nodes.go:111 +0x455
```

The issue was fixed in https://github.com/kubernetes/heapster/pull/693. I'll create a patch release and bump the version.

--- a/cluster/addons/cluster-monitoring/google/heapster-controller.yaml
+++ b/cluster/addons/cluster-monitoring/google/heapster-controller.yaml
@@ -7,26 +7,26 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: heapster-v11
+  name: heapster-v12
   namespace: kube-system
   labels:
     k8s-app: heapster
-    version: v11
+    version: v12
     kubernetes.io/cluster-service: "true"
 spec:
   replicas: 1
   selector:
     k8s-app: heapster
-    version: v11
+    version: v12
   template:
     metadata:
       labels:
         k8s-app: heapster
-        version: v11
+        version: v12
         kubernetes.io/cluster-service: "true"
     spec:
       containers:
-        - image: gcr.io/google_containers/heapster:v0.18.4
+        - image: gcr.io/google_containers/heapster:v0.18.5
           name: heapster
           resources:
             # keep request = limit to keep this container in guaranteed class

--- a/cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml
+++ b/cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml
@@ -7,26 +7,26 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: heapster-v11
+  name: heapster-v12
   namespace: kube-system
   labels:
     k8s-app: heapster
-    version: v11
+    version: v12
     kubernetes.io/cluster-service: "true"
 spec:
   replicas: 1
   selector:
     k8s-app: heapster
-    version: v11
+    version: v12
   template:
     metadata:
       labels:
         k8s-app: heapster
-        version: v11
+        version: v12
         kubernetes.io/cluster-service: "true"
     spec:
       containers:
-        - image: gcr.io/google_containers/heapster:v0.18.4
+        - image: gcr.io/google_containers/heapster:v0.18.5
           name: heapster
           resources:
             # keep request = limit to keep this container in guaranteed class

--- a/cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml
+++ b/cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml
@@ -7,26 +7,26 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: heapster-v11
+  name: heapster-v12
   namespace: kube-system
   labels:
     k8s-app: heapster
-    version: v11
+    version: v12
     kubernetes.io/cluster-service: "true"
 spec:
   replicas: 1
   selector:
     k8s-app: heapster
-    version: v11
+    version: v12
   template:
     metadata:
       labels:
         k8s-app: heapster
-        version: v11
+        version: v12
         kubernetes.io/cluster-service: "true"
     spec:
       containers:
-        - image: gcr.io/google_containers/heapster:v0.18.4
+        - image: gcr.io/google_containers/heapster:v0.18.5
           name: heapster
           resources:
             # keep request = limit to keep this container in guaranteed class

--- a/cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml
+++ b/cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml
@@ -7,26 +7,26 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: heapster-v11
+  name: heapster-v12
   namespace: kube-system
   labels:
     k8s-app: heapster
-    version: v11
+    version: v12
     kubernetes.io/cluster-service: "true"
 spec:
   replicas: 1
   selector:
     k8s-app: heapster
-    version: v11
+    version: v12
   template:
     metadata:
       labels:
         k8s-app: heapster
-        version: v11
+        version: v12
         kubernetes.io/cluster-service: "true"
     spec:
       containers:
-        - image: gcr.io/google_containers/heapster:v0.18.4
+        - image: gcr.io/google_containers/heapster:v0.18.5
           name: heapster
           resources:
             # keep request = limit to keep this container in guaranteed class

